common/Cargo.toml

```diff
@@ -9,16 +9,13 @@ publish = false
 [dependencies]
 anyhow = "1.0.31"
 async-trait = "0.1.50"
-eip-712-derive = { git = "https://github.com/graphprotocol/eip-712-derive" }
 slog = { version = "2.5.2", features = ["release_max_level_trace", "max_level_trace"] }
 slog-term = "2.6.0"
 slog-async = "2.5.0"
 futures = { version="0.3.5", features=["compat"] }
 warp = "0.3.1"
 prometheus = "0.12.0"
-tokio = { version="1.10", features=["sync", "time"] }
-web3 = "0.16.0"
-solidity-bindgen = "0.1.0"
+tokio = { version="1.10", features=["sync", "time", "rt-multi-thread"] }
```

Comment:
This can be a separate PR: probably a good idea to upgrade these dependencies. I typically check for the latest versions with [cargo-outdated](https://github.com/kbknapp/cargo-outdated) (`cargo outdated -R`)

Reply:
Definitely, I'll do it as a new PR.

<img width="633" alt="Screenshot 2024-02-08 at 14 51 40" src="https://github.com/graphprotocol/subgraph-oracle/assets/86025070/d1fac3a5-73dd-47f2-b050-652444c45a73">